### PR TITLE
DEVDOCS-4325: add external_order_id to v2/orders

### PR DESCRIPTION
**Pull request by danielquachnz**
# [DEVDOCS-4325](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4325)

## What changed?
* Add `external_order_id` to `v2/orders`

@bigcommerce/orders

### CODE REVIEW

**Review by WillemHoman — DISMISSED** [review]

**Comment by bc-vincent-zhao** [comment]
@bigcommerce/dev-docs would you please review this change ? thanks

File: reference/orders.v2.oas2.yml [hunk 1]
```diff
@@ -3075,9 +3079,13 @@ components:
...
                   - ''
               customer_locale:
-                type: 'string,'
+                type: string
                 example: en
                 description: The customer’s locale.
+              external_order_id:
+                type: string
+                example: 'external-order-id'
+                description: The external id of the order
```
Comment (bc-tgomez) [inline reference/orders.v2.oas2.yml:3088]:
```suggestion
                description: The external id of the order.
```

File: reference/orders.v2.oas2.yml [hunk 2]
```diff
@@ -4216,6 +4224,10 @@ components:
           type: string
           example: en
           description: The customer’s locale.
+        external_order_id:
+          type: string
+          example: 'external-order-id'
+          description: The external id of the order
```
Comment (bc-tgomez) [inline reference/orders.v2.oas2.yml:4230]:
```suggestion
          description: The external id of the order.
```

**Review by bc-tgomez — APPROVED** [review]
LGTM 👍

**Review by traci707 — COMMENTED** [review]
Looks fine, it might add some value to use a dummy value.

**Comment by danielquachnz** [comment]
> Looks fine, it might add some value to use a dummy value.

Hey @traci707. Happy to make this change, but do you mind providing me with an example? Do you mean replacing the blank strings with 'external-order-id'?

**Comment by traci707** [comment]
@danielquachnz I was thinking we could use a fake `external order id` number. Like the `order_id` used in [Get Accepted Payment Method](https://developer.bigcommerce.com/api-reference/ad49e43ce0a38-get-accepted-payment-methods) endpoint. 
{
"order_id": 155
}

**Comment by danielquachnz** [comment]
> @danielquachnz I was thinking we could use a fake `external order id` number. Like the `order_id` used in the [Get Accepted Payment Method](https://developer.bigcommerce.com/api-reference/ad49e43ce0a38-get-accepted-payment-methods) endpoint. { "order_id": 155 }

Hey @traci707 , I have some dummy values as 'external-order-id'. We are accepting a string here as we don't know the format of the external order id. Happy for me to replace all the blanks (`''`) with 'external-order-id'?

**Comment by traci707** [comment]
@danielquachnz that's fine

**Comment by bc-tgomez** [comment]
@markcmurphy @slsriehl Could ya'll approve this merge for the Orders team please?

**Review by markcmurphy — APPROVED** [review]
Sorry for the delay, @danielquachnz. Thanks for this!

**Pull request merged**